samples/reducer.js

```diff
@@ -104,10 +104,9 @@ function createReducer (config) {
           ...state.records,
           [id]: action.payload
         }
-      // TODO: here is a case where ordering might be nice. only save meta after created
```

Comment:
from testing, the state of effects seem to operate in order. so in this case, `createMeta` adds a `createdAt` field which `saveMeta` then is able to see in the store

note i converted `saveMeta` to just take an `id` so it reads the most recently updated version from the store